certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/message/dto/QuestionDTO.java

```diff
@@ -20,7 +22,7 @@ public class QuestionDTO {
   String message;
   String author;
   LocalDateTime sent;
-  ComplementDTO[] complementDTOS;
+  ComplementDTO[] complements;
```

Comment:
Lets change it to List instead of array.. makes it easier for us to code with it... Same for other string-arrays.

I should have made the change when introducing the DTO´s from common, but missed it.